src/styles/main.scss

```diff
@@ -0,0 +1,86 @@
+$day-size: 100px;
+$border-color: #000;
+$backgroundColor: #eee;
+$backgroundColorHover: #ffbfcb;
+$border-width: 1px;
+$gap-size: 1px;
+$padding-size: 10px;
+$font-size: 30px;
+$font-family: Arial, Helvetica, sans-serif;
+$moveUp: -20px;
+$calendar-width: calc(
+  (#{$day-size} * 7) + (#{$gap-size} * 6) + (#{$padding-size})
+);
+$calendar_week: (
```

Comment:
Variable names should be written in kebab-case. Please change `$calendar_week` to `$calendar-week` to adhere to the code style rules.